utilities_common/sfp_helper.py

```diff
@@ -19,6 +19,43 @@
     'application_advertisement': 'Application Advertisement'
 }
 
+CMIS_DATA_MAP = {
```

Comment:
CMIS_DATA_MAP seems to be just a super set of QSFP_DATA_MAP. 
Would it be better to only list the CMIS specific data here (the delta), then later get the entire cmis_data_map by doing QSFP_DATA_MAP.update(CMIS_SPECIFIC_DATA_MAP)?  (the naming for CMIS_SPECIFC_DATA_MAP might not be perfect, you may choose whatever naming you feel suited it better)

Reply:
Yes, have modified based on this now.